charts/apiclarity/templates/deployment.yaml

```diff
@@ -76,11 +76,13 @@ spec:
               value: {{ .Values.apiclarity.notificationURLPrefix }}
             {{- end }}
             - name: FUZZER_DEPLOYMENT_TYPE
-              value: kubernetes
+              value: {{ .Values.apiclarity.env.plugins.FUZZER_DEPLOYMENT_TYPE }}
             - name: FUZZER_IMAGE_NAME
               value: {{ .Values.apiclarity.modules.fuzzer.docker.image }}
             - name: FUZZER_PLATFORM_HOST_FROM_FUZZER
               value: http://{{ include "apiclarity.name" . }}.{{ .Release.Namespace }}:8080/api
```

Comment:
`FUZZER_IMAGE_NAME` and `FUZZER_PLATFORM_HOST_FROM_FUZZER` will now be configured on the config map template, right? so no need it here?

Reply:
No need in case of configmap use, right. FUZZER_IMAGE_NAME can be removed from this file (must be used only when starting ApiClarity in command line) FUZZER_PLATFORM_HOST_FROM_FUZZER can be moved to configmap